app/settings.py

```diff
@@ -30,3 +32,7 @@ def get_key(key_name):
 FTP_FOLDER = os.getenv('SEFT_CONSUMER_FTP_FOLDER', '.')
 
 SDX_KEYS_FILE = os.getenv('SDX_KEYS_FILE', 'keys.yml')
+
+# Configure the number of retries attempted before failing call
+TOTAL_RETRIES = 5
```

Comment:
Could do with more context - total retries of what? Receipting? Consuming? Something else?